FlowCrypt/Functionality/Services/Contacts Services/LocalContactsProvider.swift

```diff
@@ -61,21 +58,37 @@ extension LocalContactsProvider: LocalContactsProviderType {
     }
 
     func searchContact(with email: String) -> Contact? {
+        guard let contactObject = find(with: email) else { return nil }
+        return Contact(contactObject)
+    }
```

Comment:
This method should be probably updated to return `[Contact]`, renamed to `searchContacts`

Reply:
`Contact` is not a good name here, actually. Let's call it, for now, `RecipientPubKey` for lack of a better name.

And yup - there may be more than one RecipientPubKey per recipient, which feels more natural to say now that we renamed it.